### ISSUE TITLE
Error code 6

### ISSUE THREAD

**Issue by Lloydplays** [minecraft-linux/mcpelauncher-manifest#566]
randomly crashes

**Comment by selma31000** [comment]
> se bloque au hasard

oui sa me fait la meme chose

**Comment by selma31000** [comment]
> se bloque au hasard

oui sa me fait la meme chose

**Comment by Lloydplays** [comment]
Yeah it happens on Mac I haven't tried it on Linux but I'll try today soon because I recently got ubuntu on a different computer
